checka11y.css

```diff
@@ -65,3 +65,12 @@ nav + nav::after {
   content: "ERROR: <nav> denotes the primary navigation for the page. Other navigations are permitted but should not be wrapped in a <nav>" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+/**
+ * Links
+ */
+
+ a[target='_blank']::after /* Links that open a new page/tab target="_blank" */ {
+  content: " ERROR: Ideally, avoid links that open in a new tab or window. If a link does, ensure the link's behavior will be communicated in a way that is apparent to all users" !important;
```

Comment:
Maybe something like "WARNING: Links that open in a new tab or window should be communicated to the user. Ensure the user knows of this behaviour."

This way we avoid the word "avoid".